projects/packages/forms/changelog/update-move-contact-form-static-assets

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I don't think "minor" is intended for this kind of things :D

Reply:
It's not breaking compatibility, or doing much really. Just adding some files 🙂